openstudiocore/src/osversion/VersionTranslator.hpp

```diff
@@ -162,6 +162,7 @@ class OSVERSION_API VersionTranslator {
   int m_nObjectsFinalIdf;
   int m_nObjectsFinalModel;
   bool m_isComponent;
+  std::vector<IdfObject> m_cbeccSizingObjects;
```

Comment:
well the very idea of what I'm doing here is pretty horrific in my mind, but I think this is decent considering the options. Users should be none the wiser and it should just work for them.